### DISCUSSION TITLE
Who owns the rights of the created image?

### DISCUSSION THREAD

**Discussion by Efe-Cal** [lllyasviel/Fooocus#1573] [Q&A]
Who owns the right? Can I sell the images?

**Comment by Hypersonic-Popcorn** [comment]
That's going to depend on what country you're dealing with, maybe whether you've modified the image significantly, and possibly other factors. You may want to talk to a lawyer.

**Comment by mashb1t** [comment]
SDXL (not Turbo):

"each Contributor hereby grants
to You a perpetual, worldwide, non-exclusive, no-charge, royalty-free, irrevocable
(except as stated in this paragraph) patent license to make, have made, use, offer to
sell, sell, import, and otherwise transfer the Model and the Complementary Material"
Source: https://github.com/Stability-AI/generative-models/blob/main/model_licenses/LICENSE-SDXL1.0

tl;dr you own the image, and dependent on the model derivative/fine-tune (if you use one and not the SAI SDXL base model) you may or may not sell the images depending on their license terms.

DISCLAIMER: i am not a lawyer, please read the provided license thoroughly and if uncertain, contact a lawyer.
